code/espurna/sensor.ino

```diff
@@ -42,6 +42,10 @@ unsigned char _sensor_temperature_units = SENSOR_TEMPERATURE_UNITS;
 double _sensor_temperature_correction = SENSOR_TEMPERATURE_CORRECTION;
 double _sensor_humidity_correction = SENSOR_HUMIDITY_CORRECTION;
 
+#if PZEM004T_SUPPORT
+PZEM004TSensor *pzem004t_sensor;
```

Comment:
I know this is not a good way of doing this, especially in the future when the firmware will instantiate the sensors in runtime. But I tried to avoid a for loop in sensors to get the one with SENSOR_PZEM004T_ID. But if you guys think it is better I'll change the PR.